src/test/java/org/fogbowcloud/manager/OrderStateTransitionerTest.java

```diff
@@ -84,73 +93,95 @@ public void testValidTransition() throws UnexpectedException {
         Order order = createOrder(originState);
         openOrdersList.addItem(order);
 
+        // exercise
         OrderStateTransitioner.transition(order, destinationState);
-        assertNull(openOrdersList.getNext());
-        assertEquals(order, spawningOrdersList.getNext());
+
+        // verify
```

Comment:
it would be nice to assert is was different before the exercise

Reply:
Fixed.